motor_selective_stop_signal/experiment.js

```diff
@@ -1,6 +1,18 @@
 /* ************************************ */
 /* Define helper functions */
 /* ************************************ */
+function median(values) {
+
+    values.sort( function(a,b) {return a - b;} );
+
+    var half = Math.floor(values.length/2);
+
+    if(values.length % 2)
+        return values[half];
+    else
+        return (values[half-1] + values[half]) / 2.0;
+}
+
```

Comment:
Instead of writing helper functions for simple operations like this (likely to be error prone, etc) I suggest [using a library.](http://mathjs.org/index.html) for example, with mathjs:

      math.median([1,2,3,4,5])
      3

These functions should generally be available for all experiments.